src/integrationTest/java/uk/gov/hmcts/reform/bulkscanprocessor/entity/ZipFilesSummaryRepositoryTest.java

```diff
@@ -0,0 +1,123 @@
+
+        dbHasEvents(
+            event("c1", "test1.zip", createdDate, ZIPFILE_PROCESSING_STARTED),
+            event("c1", "test1.zip", completedDate, Event.COMPLETED),
+            event("c2", "test2.zip", createdDate.minus(1, MINUTES), ZIPFILE_PROCESSING_STARTED),
+            event("c4", "test4.zip", createdDate.minus(1, HOURS), ZIPFILE_PROCESSING_STARTED),
+            event("c4", "test4.zip", createdDate.minus(30, MINUTES), FILE_VALIDATION_FAILURE)
+        );
+
+        // when
+        List<ZipFileSummaryItem> result = reportRepo.getZipFileSummaryReportFor(LocalDate.of(2019, 2, 15));
+
+        // then
+        assertThat(result)
+            .usingFieldByFieldElementComparator()
+            .containsExactlyInAnyOrder(
+                new Item(
+                    "test1.zip", createdDate, completedDate, "c1", COMPLETED.toString()
+                ),
+                new Item(
+                    "test2.zip", createdDate.minus(1, MINUTES), null, "c2", ZIPFILE_PROCESSING_STARTED.toString()
+                ),
+                new Item("test4.zip", createdDate.minus(1, HOURS), null, "c4", FILE_VALIDATION_FAILURE.toString())
+            );
+    }
+
+    @Test
+    public void should_return_empty_zipfile_summary_when_no_zipfiles_received() {
+        // given
+        Instant createdAt = Instant.parse("2019-02-09T14:15:23.456Z");
+
+        dbHasEvents(
+            event("c1", "test1.zip", createdAt, Event.ZIPFILE_PROCESSING_STARTED),
+            event("c1", "test1.zip", createdAt.minus(2, MINUTES), Event.FILE_VALIDATION_FAILURE),
+            event("c2", "test2.zip", Instant.parse("2019-02-09T14:15:23.456Z"), Event.ZIPFILE_PROCESSING_STARTED),
+            event("c2", "test2.zip", Instant.parse("2019-02-10T14:15:23.456Z"), Event.COMPLETED)
+        );
+
+        // when
+        List<ZipFileSummaryItem> result = reportRepo.getZipFileSummaryReportFor(LocalDate.of(2019, 2, 10));
+
+        // then
+        assertThat(result).isEmpty();
+    }
+
+    @Test
+    public void should_return_zipfilesummary_for_the_requested_date_when_completed_date_is_not_same_as_createdDate() {
+        // given
+        Instant createdAt = Instant.parse("2019-02-15T23:59:23.456Z");
+        Instant nextDay = Instant.parse("2019-02-16T00:00:23.456Z");
+
+        dbHasEvents(
+            event("c1", "test1.zip", createdAt, Event.ZIPFILE_PROCESSING_STARTED),
+            event("c1", "test1.zip", nextDay, Event.COMPLETED),
+            event("c1", "test2.zip", nextDay, Event.ZIPFILE_PROCESSING_STARTED)
+        );
+
+        // when
+        List<ZipFileSummaryItem> result = reportRepo.getZipFileSummaryReportFor(LocalDate.of(2019, 2, 15));
+
+        // then
+        assertThat(result)
+            .usingFieldByFieldElementComparator()
+            .containsExactlyInAnyOrder(
+                new Item(
+                    "test1.zip", createdAt, nextDay, "c1", ZIPFILE_PROCESSING_STARTED.toString()
```

Comment:
should be `COMPLETED`

Reply:
@rkondratowicz tests should be ok now :)